src/api/server/configuration.rst

```diff
@@ -16,316 +16,14 @@
 Configuration
 =============
 
-The CouchDB Server Configuration API provide an interface to query and update
-the various configuration values within a running CouchDB instance.
+The classical CouchDB Server Configuration API located at ``/_config`` will
+return a 404 in CouchDB 2.0 and got replaced by :ref:`HTTP API <api/node>`.
 
-``/_config``
```

Comment:
You cannot just limit yourself here with `_config`, `_compaction`, `_stats` etc. because even similar endpoints may return different responses:

`GET http://localhost:15984` :
```
{
    "couchdb": "Welcome",
    "vendor": {
        "name": "The Apache Software Foundation"
    },
    "version": "f29e590"
}
```

`GET http://localhost:15986` :
```
{
    "couchdb": "Welcome",
    "uuid": "d338f1c4b4555c9c7ef6ad5e8ec34667",
    "vendor": {
        "name": "The Apache Software Foundation"
    },
    "version": "f29e590"
}

```

`GET http://localhost:15984/_users`:
```
{
    "compact_running": false,
    "data_size": 0,
    "db_name": "_users",
    "disk_format_version": 6,
    "disk_size": 33936,
    "doc_count": 0,
    "doc_del_count": 0,
    "instance_start_time": "0",
    "other": {
        "data_size": 0
    },
    "purge_seq": 0,
    "sizes": {
        "active": 0,
        "external": 0,
        "file": 33936
    },
    "update_seq": [
        0,
        "g1AAAAFDeJzLYWBg4MhgTmHgz8tPSTV0MDQy1zMAQsMcoARTIkOS_P___7MSGXAqSVIAkkn2hFQ5gFTFE1KVAFJVT0BVHguQZGgAUkCF8wmrXABRuZ-wygMQlfcJq3wAUQlyZxYAHlFWdg"
    ]
}

```

`GET http://localhost:15986/_users`:
```
{
    "committed_update_seq": 1,
    "compact_running": false,
    "compacted_seq": 0,
    "data_size": 2196,
    "db_name": "_users",
    "disk_format_version": 6,
    "disk_size": 8376,
    "doc_count": 1,
    "doc_del_count": 0,
    "instance_start_time": "1436359348837084",
    "other": {
        "data_size": 2000
    },
    "purge_seq": 0,
    "sizes": {
        "active": 2196,
        "external": 2000,
        "file": 8376
    },
    "update_seq": 1,
    "uuid": "07df00f7940efd8e908b47c1cbb8f3f5"
}
```